src/common.js

```diff
@@ -134,7 +134,7 @@ export function parseImportMap (json, baseUrl) {
       let resolvedScopeName = resolveUrl(scopeName, baseUrl);
```

Comment:
Can we remove the entire resolvedScopeName variable? Looks like it’s dead/unused?

Reply:
Oops I misread this — disregard 😀